### ISSUE TITLE
correct release tag? error: The 'pubs==0.8.dev2' distribution was not found and is required by the application

### ISSUE THREAD

**Issue by beuerle** [pubs/pubs#156]
Is pubs steering towards singularity? The new release is `v0.6.0`, instead of `v0.8.0`.

![2018-08-07_11 47 17](https://user-images.githubusercontent.com/10122039/43768449-b4d70a62-9a37-11e8-8e2c-0efc7367ab35.png)

Most probably related, but I didn't have time yet to dig into it; when installing the most recent revision of `pubs` (I'm on Arch, installing `pubs-git` from AUR) I get the following error message now:

```bash
$ pubs
Traceback (most recent call last):
  File "/bin/pubs", line 6, in <module>
    from pkg_resources import load_entry_point
  File "/usr/lib/python3.7/site-packages/pkg_resources/__init__.py", line 3095, in <module>
    @_call_aside
  File "/usr/lib/python3.7/site-packages/pkg_resources/__init__.py", line 3079, in _call_aside
    f(*args, **kwargs)
  File "/usr/lib/python3.7/site-packages/pkg_resources/__init__.py", line 3108, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/usr/lib/python3.7/site-packages/pkg_resources/__init__.py", line 570, in _build_master
    ws.require(__requires__)
  File "/usr/lib/python3.7/site-packages/pkg_resources/__init__.py", line 888, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/usr/lib/python3.7/site-packages/pkg_resources/__init__.py", line 774, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'pubs==0.8.dev2' distribution was not found and is required by the application
```

**Comment by omangin** [comment]
Hi @beuerle. Regarding the version, what happened is that, for documentation purpose, I generated the git tag for the version `0.6.0` so Github assumes the version was released at that date while it is actually older but no tag was generated at that time.

Regarding your issue I cannot reproduce it. Did you try the latest version of the AUR's `PKGBUILD`? I am not certain of where your issue comes from but you might have to report it to the `PKGBUILD` maintainer too.
